src/libponyrt/actor/actor.h

```diff
@@ -91,6 +80,8 @@ gc_t* ponyint_actor_gc(pony_actor_t* actor);
 
 heap_t* ponyint_actor_heap(pony_actor_t* actor);
 
+bool ponyint_actor_unscheduled(pony_actor_t* actor);
```

Comment:
note: this will be removed if the `FLAG_UNSCHEDULED` logic is removed as is being discussed in https://github.com/ponylang/ponyc/pull/4137